src/binder/statement/insert_statement.cpp

```diff
@@ -1,10 +1,29 @@
+#include "fmt/ranges.h"
+
+#include "binder/bound_expression.h"
+#include "binder/bound_table_ref.h"
 #include "binder/statement/insert_statement.h"
+#include "binder/statement/select_statement.h"
+#include "common/util/string_util.h"
 
 namespace bustub {
 
-InsertStatement::InsertStatement(duckdb_libpgquery::PGInsertStmt *pg_stmt)
-    : BoundStatement(StatementType::INSERT_STATEMENT) {}
+auto IndentAllLines(const std::string &lines, size_t num_indent) -> std::string {
```

Comment:
This seems like code that belongs in StringUtil

Reply:
+1